### PR TITLE
Update package dependencies

### PR DESCRIPTION
**Pull request by notlmn**
- Update `element-ready` to `4.1.0` from `3.1.0` to stop rAF from running after page load ([context](https://github.com/sindresorhus/refined-github/issues/2290#issuecomment-516846928)).
- Updated rest of the packages too.
- Add `"private": true` to `package.json` to avoid warning message while using `npm install`.
- Copy over `tsconfig.json` from RGH.

### CODE REVIEW

**Comment by sindresorhus** [comment]
@fregante Good to merge?

File: package.json [hunk 1]
```diff
@@ -8,22 +9,32 @@
 		"release": "run-s build update-version release:*",
 		"update-version": "dot-json distribution/manifest.json version $TRAVIS_TAG"
 	},
+	"dependencies": {
+		"@types/chrome": "0.0.88",
+		"dom-chef": "^3.6.1",
+		"element-ready": "^4.1.0",
+		"select-dom": "^5.1.0",
+		"webext-domain-permission-toggle": "^1.0.0",
+		"webext-dynamic-content-scripts": "^6.0.2"
```
Comment (fregante) [inline package.json]: This should be the `next` tag:

```suggestion
		"webext-dynamic-content-scripts": "^6.0.3-0"
```

A bugfix was sent but I haven't published the final version yet.

Comment (notlmn) [reply package.json]: Done!

File: tsconfig.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
 	"extends": "@sindresorhus/tsconfig",
 	"compilerOptions": {
 		"target": "esnext",
-		"module": "es2015",
```
Comment (fregante) [inline tsconfig.json]: I think this will increase the bundle size because ModuleConcatenationPlugin will stop working

**Pull request merged**